npm_test.go

```diff
@@ -37,13 +37,14 @@ type npmTestParams struct {
 	testName      string
 	nativeCommand string
 	// Deprecated
-	legacyCommand  string
-	repo           string
-	npmArgs        string
-	wd             string
-	buildNumber    string
-	moduleName     string
-	validationFunc func(*testing.T, npmTestParams, bool)
+	legacyCommand      string
+	repo               string
+	npmArgs            string
+	wd                 string
+	buildNumber        string
+	moduleName         string
+	validationFunc     func(*testing.T, npmTestParams, bool)
+	collectedBuildInfo bool
```

Comment:
Instead of adding this boolean to the ```npmTestParams``` struct, let's create a new test, that will only the installation of a single package.